src/main/java/com/google/sps/servlets/SortedMarkersServlet.java

```diff
@@ -49,41 +50,34 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
-
+ 
   /**
    * Get the calendar events for the specific group
    * Convert the events to EventMarker objects
    */
   private Collection<EventMarker> getEvents(String groupId) {
-
-      // All hardcoded tests for the moment 
-      // Will call the calendar event function to get the events for the 
-      // given groupId once merged
-      ArrayList<EventMarker> events = new ArrayList<EventMarker>(); 
-        Date date = new Date(2020-1900,8,17,18,30);
-        Date date2 = new Date(2020-1900,7,21,1,30);
-        Date date3 = new Date(2020-1900,7,31,12,30);
-        EventMarker event = new EventMarker("Rally at City Hall", 
-            "Join us at San Jose City Hall to rally for the BLM Movement!", 
-            "200 E Santa Clara St, San Jose, CA 95113", 
-            new DateTime(date), 
-            "Black Lives Matter");
-        EventMarker event2 = new EventMarker("Peaceful Protest at the Lake", 
-            "We're having a peaceful protest at Almaden Lake, come and support!", 
-            "6099 Winfield Blvd, San Jose, CA 95120", 
-            new DateTime(date2), 
-            "Black Lives Matter");
-        EventMarker event3 = new EventMarker("Beach BBQ", 
-            "Come join us in Santa Cruz to talk and enjoy eachother's company during this time.", 
-            "2320 W Cliff Dr, Santa Cruz, CA 95060", 
-            new DateTime(date3), 
-            "Black Lives Matter");
-
-        events.add(event);
-        events.add(event2);
-        events.add(event3);
-
-        SortedMarkers sorted = new SortedMarkers(events); 
-        return sorted.getSortedMarkers();
+    try {
+        ArrayList<EventMarker> mapEvents = new ArrayList<EventMarker>();
+        EventsServlet eventsServlet = new EventsServlet();
+        
+        // CalendarID and group Name are both hardcoded for now
```

Comment:
Can you rephrase this to something like TODO: Replace hard-coded calendarID and group name when ____

That way it's more clear to the reader what you're waiting for